### PR TITLE
Removed non existent countries.

### PR DESCRIPTION
**Pull request by setfenv**
Keeping fake countries may allow users to download SCRIPTWARE.

### CODE REVIEW

**Comment by FungusGenerator** [comment]
who tf making pull requests in ohio

**Comment by LuauSource** [comment]
Remove Ohio

**Comment by GalacticInspired** [comment]
Why did you remove ukraine?

**Comment by DimensionalFusion** [comment]
WHAT A GIGACHAD PULL REQUEST

**Review by DimensionalFusion — APPROVED** [review]

**Comment by setfenv** [comment]
> Why did you remove ukraine?

Complains about children dying because Russia bombs them but then proceeds to fund Israel's terrorist group "IDF"

**Review by GalacticInspired — COMMENTED** [review]
why did you remove countries
and ohio is a state, not a country

this is a troll pull request which is sad

**Comment by FungusGenerator** [comment]
> proceeds to fund Israel's terrorist group "IDF"

+ A country much more stronger than us whom also wields nuclear weapons is attacking us, what do we do?
- Wouldn't it be funny if we funded the Israeli Defense Forces lol
+ Wtf is wrong with you

**Comment by setfenv** [comment]
> > proceeds to fund Israel's terrorist group "IDF"
> 
> • A country much more stronger than us whom also wields nuclear weapons is attacking us, what do we do?
> 
> ~ Wouldn't it be funny if we funded the Israeli Defense Forces lol
> 
> • Wtf is wrong with you

You’re being rather schizophrenic today. Please take one Abilify. Thank you and have a nice day.

**Comment by Deniied0** [comment]
As Smellypooper3, king of the africas, I approve this message.

**Comment by FungusGenerator** [comment]
> You’re being rather schizophrenic today.

Couldn't get more schizophrenic than a mf who uses a roblox system to spew conspiracy theories

**Comment by Sceleratls** [comment]
Thanks for letting us know. We're looking into this, as a urgent security vulnerability, and we have some of our Top Staff working on it as we speak. 

However in order to aid in the investigation, further information from you is required:
- Is SCRIPTWARE a Level 7 or a Level 8 exploit?
- Does SCRIPTWARE run on M1 chip, as I use a Mac?
- Do you consider this a vital threat to the Adonis Ecosystem?

Hoping to dear back, 
Sceleratls

File: MainModule/Server/Dependencies/CountryRegionCodes.lua [hunk 1]
```diff
@@ -165,6 +164,7 @@ return {
 	NF = "Norfolk Island",
 	MP = "Northern Mariana Islands",
 	NO = "Norway",
+	OH = "Ohio",
```
Comment (Sceleratls) [inline MainModule/Server/Dependencies/CountryRegionCodes.lua:167]: Is this needed to improve performance in the Luau?

**Pull request closed without merging**